Batch_Json_files/.gitignore

```diff
@@ -1,4 +0,0 @@
-# This directory is intentionarly ignored - this file only applies to the git repo
```

Comment:
Can this file be added back in? It's important for testing and stops any batch.json files from accidentally being uploaded to main

Reply:
Oops... Sorry 'bout that. Accidentally deleted when testing.

Reply:
All good! Just added it back in another commit